docs/modules/ROOT/pages/libraries-crypto.adoc

```diff
@@ -1,5 +1,35 @@
 ## crypto
 
+### `decrypt(string value, string secret, string algorithm, string mode, string padding)`
```

Comment:
docs need to be updated with your changes to the functions

Reply:
ah yeah